tfjs-layers/src/engine/training.ts

```diff
@@ -330,10 +330,10 @@ function checkInputData(
         const refDim = shapes[i][j];
         if (refDim != null) {
           if (refDim !== dim) {
-            throw new ValueError(
-                `Error when checking ${exceptionPrefix}: expected ` +
-                `${names[i]} to have shape ${JSON.stringify(shapes[i])} but ` +
-                `got array with shape ${JSON.stringify(array.shape)}.`);
+              throw new ValueError(
+                `${names[i]} is expected to have an input shape of [${shapes[i]}] ` +
```

Comment:
this line is too long (>80 chars) which is why the CI is failing. Can you make sure it's less than 80 chars then I can merge? Thanks!

Reply:
@nsthorat : fixed it :smile: